native/src/components/SprungbrettOffer.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import {
+  createSprungbrettJobsEndpoint,
+  OfferModel,
+  SprungbrettJobModel,
+  useLoadFromEndpoint,
+} from '../../../api-client/src'
+import useSnackbar from '../hooks/useSnackbar'
+import LoadingErrorHandler from '../routes/LoadingErrorHandler'
+import openExternalUrl from '../utils/openExternalUrl'
+import List from './List'
+import SprungbrettListItem from './SprungbrettListItem'
+
+type SprungbrettOfferProps = {
+  sprungbrettOffer: OfferModel
+  languageCode: string
+}
+
+const SprungbrettOffer = ({ sprungbrettOffer, languageCode }: SprungbrettOfferProps): ReactElement => {
```

Comment:
so this code is identical to `routes/SprungbrettOffer.tsx` besides the useLoadFromEndpoint and the error handler? To me it seems this should be the route and the other one the component. 
But also probably never mind as we'll remove one of those files in the next few weeks anyway...

Reply:
Yes, routes/SprungbrettOffer.tsx will be removed in #2637. I disagree, since there will not be a standalone offers/sprungbrett offer route anymore, offers will only ever be shown embedded in a category. Therefore, it should be a component and not a route/container. This also makes sense since all meta tags/language change paths and so on should be set by the category page already.